docs/comparison.md

```diff
@@ -275,3 +272,76 @@ that don't have equivalents in the OpenTelemetry Collector:
+| `operators`     | []       | An array of [operators](https://github.com/open-telemetry/opentelemetry-log-collection/blob/main/docs/operators/README.md#what-operators-are-available). See below for more details                                                            |
+| `raw`           | false    | If true, the windows events are not processed and sent as XML.                                                                                                                                                                                 |
+| `storage`       | none     | The ID of a storage extension to be used to store bookmarks. Bookmarks allow the receiver to pick up where it left off in the case of a collector restart. If no storage extension is used, the receiver will manage bookmarks in memory only. |
+
+For more information, please refer to the [windows event log receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/windowseventlogreceiver/README.md) readme.
+
+#### Windows Performance Counters Receiver
+
+This receiver, for Windows only, captures the configured system, application, or custom performance counter data from the Windows registry using the [PDH interface](https://learn.microsoft.com/en-us/windows/win32/perfctrs/using-the-pdh-functions-to-consume-counter-data). It is based on the [Telegraf Windows Performance Counters Input Plugin](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/win_perf_counters)
+
+- `Memory\Committed Bytes`
+- `Processor\% Processor Time`, with a datapoint for each `Instance` label = (`_Total`, `1`, `2`, `3`, ... )
+
+If one of the specified performance counters cannot be loaded on startup, a
+warning will be printed, but the application will not fail fast. It is expected
+that some performance counters may not exist on some systems due to different OS
+configuration.
+
+##### Configuration
+
+The collection interval and the list of performance counters to be scraped can
+be configured:
+
+```yaml
+windowsperfcounters:
+  collection_interval: <duration> # default = "1m"
+  metrics:
+    <metric name>:
+      description: <description>
+      unit: <unit type>
+      gauge:
+    <metric name>:
+      description: <description>
+      unit: <unit type>
+      sum:
+        aggregation: <cumulative or delta>
+        monotonic: <true or false>
+  perfcounters:
+    - object: <object name>
+      instances: [<instance name>]*
+      counters:
+        - name: <counter name>
+          metric: <metric name>
+          attributes:
+            <key>: <value>
+```
+
+For more information, please refer to the [windows performance counter receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/windowsperfcountersreceiver/README.md) readme.
+
+*Note: The following sources are not supported by the OpenTelemetry collector yet:
```

Comment:
```suggestion
*Note:* The following sources are not supported by the OpenTelemetry collector yet:
```

What does it actually mean? For me it sounds like we do not support it but listing here, which doesn't make much sense

Reply:
I think we should put them it in the table and it will be better

Reply:
It makes sense when I re-read it, but it would be nice to add it to the comparison table as well :)